plugin/src/main/groovy/grails/plugin/springsecurity/ui/strategy/DefaultRegistrationCodeStrategy.groovy

```diff
@@ -40,6 +41,14 @@ class DefaultRegistrationCodeStrategy implements RegistrationCodeStrategy {
 		springSecurityUiService.register user, password, salt
 	}
 
+	def verifyRegistration(String token) {
```

Comment:
Why was verification moved out of the controller and into the default strategy?

Reply:
See comment below.